spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnSelector.scala

```diff
@@ -2,9 +2,19 @@ package com.datastax.spark.connector
 
 import scala.language.implicitConversions
 
-sealed trait ColumnSelector
-case object AllColumns extends ColumnSelector
-case class SomeColumns(columns: NamedColumnRef*) extends ColumnSelector
+sealed trait ColumnSelector {
+  def toAliasMap: Map[String, String]
+}
+
+case object AllColumns extends ColumnSelector {
+  override def toAliasMap: Map[String, String] = Map.empty
```

Comment:
I could write something like `Map.empty.withDefault(x => x)` but this is not the same. Later in the code, there is a test whether the alias is defined for the field. If it is not, the convention is used. If I implemented the default identity, I wouldn't be able to fallback to convention.